### PR TITLE
[tw] Localize `Microservices Architecture` for Traditional Chinese

### LINKED ISSUE #2476: [tw] Localize `Microservices Architecture` for Traditional Chinese
1. Localize `Microservices Architecture` for Traditional Chinese
2. https://glossary.cncf.io/microservices-architecture/

### PR DESCRIPTION
**Pull request by EmilyXinyi**
### Describe your changes
original content from https://glossary.cncf.io/microservices-architecture/

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2476 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/microservices-architecture.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: 微服務架構
+status: Completed
+category: 概念
+tags: ["架構", "基本原理", ""]
+---
+
+## 是什麼 {#what-it-is}
```
Comment (pichuang) [inline content/zh-tw/microservices-architecture.md:8]:
```suggestion
```

File: content/zh-tw/microservices-architecture.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+微服務架構是一種將應用程式分解為獨立的（微）[服務](/zh-tw/service/)的架構方法，每個服務都專注於特定的功能。
+這些服務密切協作，對終端使用者來説就像是單一實體。以 Netflix 為例，其界面允許您訪問、搜尋和預覽影片。
+這些功能可能由更小的服務提供支持，每個服務負責一個功能，例如身份驗證、搜尋和在瀏覽器中運行預覽。
+
+與緊密耦合的[單體式應用程式](/zh-tw/monolithic-apps/）相比，這用架構方法使開發人員能夠更快地推出新功能或更新功能。
+
+## 解決的問題 {#problem-it-addresses}
+
+應用程式由不同的部分組成，每個部分負責特定的功能。對特定功能的需求不一定會隨著應用程式其它部分的需求而增加或減少。
+回到我們的 Netflix 範例：假設在一場大規模的行銷活動之後，Netflix 的註冊人數大幅上升，但流媒體在當天凌晨的時間內或多或少保持穩定。
```
Comment (pichuang) [inline content/zh-tw/microservices-architecture.md:19]:
```suggestion
回到我們的 Netflix 範例：假設在一場大規模的行銷活動之後，Netflix 的註冊人數大幅上升，但在每天凌晨的串流影音撥放量基本保持穩定。
```

File: content/zh-tw/microservices-architecture.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+註冊人數的激增需要更多的註冊容量。用傳統的單一方法，整個應用程式必須[擴展](/zh-tw/scalability)以適應這種增長——資源的利用效率非常低。
+
+單體架構也使開發人員很容易陷入設計陷阱。
```
Comment (pichuang) [inline content/zh-tw/microservices-architecture.md:22]:
```suggestion
單體式架構也使開發人員很容易陷入設計陷阱。
```

**Review by pichuang — COMMENTED** [review]
Leave a few of comments below, and thx for contributing 👍

**Comment by EmilyXinyi** [comment]
> Leave a few of comments below, and thx for contributing 👍

感謝, 已更正

**Comment by EmilyXinyi** [comment]
> @EmilyXinyi Please squash these commits into 1 commit before merging and thanks 🙏

done, thanks!

File: content/zh-tw/microservices-architecture.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+category: 概念
+tags: ["架構", "基本原理", ""]
+---
+
+
+微服務架構是一種將應用程式分解為獨立的（微）[服務](/zh-tw/service/)的架構方法，每個服務都專注於特定的功能。
+這些服務密切協作，對終端使用者來説就像是單一實體。以 Netflix 為例，其介面允許您訪問、搜尋和預覽影片。
+這些功能可能由更小的服務提供支持，每個服務負責一個功能，例如身份驗證、搜尋和在瀏覽器中運行預覽。
+
+與緊密耦合的[單體式應用程式](/zh-tw/monolithic-apps/）相比，這用架構方法使開發人員能夠更快地推出新功能或更新功能。
```
Comment (ydFu) [inline content/zh-tw/microservices-architecture.md:13]:
```suggestion
與緊密耦合的[單體式應用程式](/zh-tw/monolithic-apps/)相比，這用架構方法使開發人員能夠更快地推出新功能或更新功能。
```

File: content/zh-tw/microservices-architecture.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+## 解決的問題 {#problem-it-addresses}
+
+應用程式由不同的部分組成，每個部分負責特定的功能。對特定功能的需求不一定會隨著應用程式其它部分的需求而增加或減少。
+回到我們的 Netflix 範例：假設在一場大規模的行銷活動之後，Netflix 的註冊人數大幅上升，但在每天凌晨的串流影音撥放量基本保持穩定。
+註冊人數的激增需要更多的註冊容量。用傳統的單一方法，整個應用程式必須[擴展](/zh-tw/scalability)以適應這種增長,故資源的利用效率非常低。
```
Comment (ydFu) [inline content/zh-tw/microservices-architecture.md:19]:
```suggestion
註冊人數的激增需要更多的註冊容量。用傳統的單一方法，整個應用程式必須[擴展](/zh-tw/scalability/)以適應這種增長,故資源的利用效率非常低。
```

**Review by ydFu — COMMENTED** [review]
miss link

File: content/zh-tw/microservices-architecture.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+單體式架構也使開發人員很容易陷入設計陷阱。
+由於所有程式碼都位於一處，因此更容易使程式碼[緊密耦合](/zh-tw/tightly-coupled-architectures/)，但更難執行關注點分離原則。
+單體應用程式通常還要求開發人員在部署任何變更之前了解整個程式碼庫。微服務架構是對這些挑戰的回應。
```
Comment (ydFu) [inline content/zh-tw/microservices-architecture.md:23]:
```suggestion
單體式應用程式通常還要求開發人員在部署任何變更之前了解整個程式碼庫。微服務架構是對這些挑戰的回應。
```
Unified translation in the same text

File: content/zh-tw/microservices-architecture.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+## 如何幫助我們 {#how-it-helps}
+
+將功能分成不同的微服務使它們更容易獨立部署、更新和擴展。
+它還允許不同的團隊同時處理較大應用程式的一小部分，而不會無意中對應用程式的其它部分產生負面影響。
+許多[雲端原生技術](/zh-tw/cloud-native-tech)旨在使微服務更易部署和管理。
```
Comment (ydFu) [inline content/zh-tw/microservices-architecture.md:29]:
```suggestion
許多[雲端原生技術](/zh-tw/cloud-native-tech/)旨在使微服務更易部署和管理。
```

**Comment by EmilyXinyi** [comment]
Just pushed a commit that addressed the above review comments and squashed previous commits

File: content/zh-tw/microservices-architecture.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+---
+title: 微服務架構
+status: Completed
+category: 概念
+tags: ["架構", "基本原理", ""]
+---
+
```
Comment (ydFu) [inline content/zh-tw/microservices-architecture.md:7]: Only one blank line is required

Comment (EmilyXinyi) [reply content/zh-tw/microservices-architecture.md:7]: 已刪除多餘空白行

File: content/zh-tw/microservices-architecture.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+這些服務密切協作，對終端使用者來説就像是單一實體。以 Netflix 為例，其介面允許您訪問、搜尋和預覽影片。
+這些功能可能由更小的服務提供支持，每個服務負責一個功能，例如身份驗證、搜尋和在瀏覽器中運行預覽。
+
+與緊密耦合的[單體式應用程式](/zh-tw/monolithic-apps/)相比，這用架構方法使開發人員能夠更快地推出新功能或更新功能。
+
+## 解決的問題 {#problem-it-addresses}
+
+應用程式由不同的部分組成，每個部分負責特定的功能。對特定功能的需求不一定會隨著應用程式其它部分的需求而增加或減少。
+回到我們的 Netflix 範例：假設在一場大規模的行銷活動之後，Netflix 的註冊人數大幅上升，但在每天凌晨的串流影音撥放量基本保持穩定。
+註冊人數的激增需要更多的註冊容量。用傳統的單一方法，整個應用程式必須[擴展](/zh-tw/scalability/)以適應這種增長,因此資源的利用效率非常低。
```
Comment (ydFu) [inline content/zh-tw/microservices-architecture.md:18]:
```suggestion
註冊人數的激增需要更多的註冊容量。用傳統的單一方法，整個應用程式必須[擴展](/zh-tw/scalability/)以適應這種增長，因此資源的利用效率非常低。
```

Comment (ydFu) [reply content/zh-tw/microservices-architecture.md:18]: also as https://github.com/cncf/glossary/pull/2655#discussion_r1414937666

**Pull request merged**